block_server.py

```diff
@@ -342,17 +342,15 @@ def write(self, reason, value):
             elif reason == BlockserverPVNames.SAVE_CONFIG:
                 with self.write_lock:
                     self.write_queue.append((self.save_active_config, (data,), "SAVING_CONFIG"))
-            elif reason == BlockserverPVNames.CLEAR_CONFIG:
-                self._active_configserver.clear_config()
-                self._initialise_config()
```

Comment:
Does this mean `CLEAR_CONFIG` is no longer used? If so, can it be safely be deleted from `PVDB`?